remappings.txt

```diff
@@ -1,6 +1,6 @@
 @openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/contracts/
 ds-test/=lib/ds-test/src/
 erc4626-tests/=lib/openzeppelin-contracts-upgradeable/lib/erc4626-tests/
-forge-std/=lib/openzeppelin-contracts-upgradeable/lib/forge-std/src/
```

Comment:
why was this changed?

Reply:
IIUC, the Forge standard library was previously being pulled from OZ. Now it's being pulled from Foundry: https://github.com/celestiaorg/blobstream-contracts/pull/322/files#diff-fe7afb5c9c916e521401d3fcfb4277d5071798c3baf83baf11d6071742823584R12

Reply:
i think i had to do this to make it work